controller/src/test/java/io/pravega/controller/server/eventProcessor/ScaleRequestHandlerTest.java

```diff
@@ -177,6 +187,17 @@ public void setup() throws Exception {
         streamStore.startUpdateConfiguration(scope, stream, config, null, executor).join();
         VersionedMetadata<StreamConfigurationRecord> configRecord = streamStore.getConfigurationRecord(scope, stream, null, executor).join();
         streamStore.completeUpdateConfiguration(scope, stream, configRecord, null, executor).join();
+
+        //Start Metrics service
+        MetricsConfig metricsConfig = MetricsConfig.builder()
```

Comment:
You are repeating the very same piece of code over and over again. Could you encapsulate this into a method that can be reused by all these tests?

Reply:
moved to a common place in a new class